src/components/Promise/RelatedFactChecks.js

```diff
@@ -26,7 +26,7 @@ function RelatedFactChecks({ factChecks, classes: classesProp }) {
 
   return (
     <div className={classes.root}>
-      {factChecks.map(({ title, date, href }) => (
+      {factChecks?.map(({ title, date, href }) => (
```

Comment:
Fact that we know we no longer need this component (or may be not in it's current form), shouldn't you be disabling it/removing it from promise page?